### PR TITLE
fix: Update fr community translation for account/ui

### LINKED ISSUE #33079: Missing community french translation in account/ui
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

translations

### Describe the bug

The community translation for account/ui are out of date

### Version

25.0.5

### Regression

- [ ] The issue is a regression

### Expected behavior

The community translation should be up to date in all supported locale

### Actual behavior

The community translation for account/ui are out of date

### How to Reproduce?

- Set the default locale to fr (French)
- Load the account/ui

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by mrleblanc101**
fix: #33079

### CODE REVIEW

**Review by ahus1 — DISMISSED** [review]
Thank you for this PR. Please add a sign-off to your commit as outlined in the CONTRIBUTING.md in this repository. Thanks!

**Comment by mrleblanc101** [comment]
@ahus1 Sorry, I used Github.dev (VS Code for the web), will add the sign-off

**Comment by ahus1** [comment]
@DimitriDR - can you please review this PR? Maybe @mrleblanc101 wants to join you for the [translations via Weblate](https://github.com/keycloak/keycloak/discussions/9270#discussioncomment-10523162).

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties [hunk 1]
```diff
@@ -169,3 +180,37 @@ unLinkSuccess=Compte dissocié avec succès
...
+verifiableCredentialsConfigAlert=Impossible de récupérer la configuration d'identification
+verifiableCredentialsOfferAlert=Impossible de récupérer une offre.
+verifiableCredentialsSelectionDefault=Sélectionnez une configuration d'identification.
+profileScopeConsentText=Profil utilisateur
+emailScopeConsentText=Courriel
+addressScopeConsentText=Adresse
+phoneScopeConsentText=Numéro de téléphone
+offlineAccessScopeConsentText=Accès hors ligne
+samlRoleListScopeConsentText=Mes roles
+rolesScopeConsentText=Roles utilisateurs
```
Comment (DimitriDR) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties:208]: Rôles avec un « ô » 😃

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties [hunk 2]
```diff
@@ -169,3 +180,37 @@ unLinkSuccess=Compte dissocié avec succès
...
+organizations=Organisations
+organizationDescription=Voir les organisations que vous avez joint
+emptyUserOrganizations=Aucune organisation
+emptyUserOrganizationsInstructions=Vous n'avez pas encore joint d'organisation.
```
Comment (DimitriDR) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties:212]: Rejoint ? Peut-être ? C'est souvent le verbe utilisé dans ces cas ? J'ai rarement vu juste "joint"

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties [hunk 3]
```diff
@@ -169,3 +180,37 @@ unLinkSuccess=Compte dissocié avec succès
...
+searchOrganization=Rechercher une organisation
+organizationList=Liste des organisations
+domains=Domaines
+refresh=Rafraichir
```
Comment (DimitriDR) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties:216]: Rafraîchir avec le petit accent qui va bien

**Review by DimitriDR — CHANGES_REQUESTED** [review]
Very clean, I just added few minor suggestions :)

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties (hunk 3 above)
Comment (mrleblanc101) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties:216]: Ah oui, je vais l'ajouté pour «refreshPage=Rafraichir la page» aussi

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties (hunk 2 above)
Comment (mrleblanc101) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties:212]: Mon québécois qui resort

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties (hunk 1 above)
Comment (mrleblanc101) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_fr.properties:208]: Parfait

**Comment by ahus1** [comment]
@DimitriDR  / @mrleblanc101 - please keep the discussion in this repository in English except where quoting a translation. This helps other contributors to follow the discussion. 

I asked for another review from @DimitriDR before this can be merged.

**Review by DimitriDR — APPROVED** [review]
Approved!

**Review by jonkoops — APPROVED** [review]
Thanks all!

**Pull request merged**